core/test/unit/helpers/foreach_spec.js

```diff
@@ -506,7 +506,7 @@ describe('{{#foreach}} helper', function () {
                         {name: 'second', visibility: 'public'},
                         {name: 'third', visibility: 'internal'},
                         {name: 'fourth', visibility: 'public'},
-                        {name: 'fifth'}
```

Comment:
Had to double check this thing as well :sweat_smile: But makes sense, especially when we have a default on db layer, model layer and even protect from setting anything else than 'public'/'internal' from the Admin API layer :+1: